config/development.config.php.dist

```diff
@@ -15,17 +15,26 @@
  * Provide your own development-mode settings by editing the file
  * `config/autoload/development.local.php.dist`.
  *
- * Because this file is aggregated last, it simply ensures:
+ * Because this file is aggregated last, it ensures:
  *
+ * - ConfigProviders for `--dev` dependencies are loaded when in development mode
+ * - Development-only overrides are applied
  * - The `debug` flag is _enabled_.
  * - Configuration caching is _disabled_.
  */
 
 declare(strict_types=1);
 
+use Laminas\ConfigAggregator\ArrayProvider;
 use Laminas\ConfigAggregator\ConfigAggregator;
+use Laminas\ConfigAggregator\PhpFileProvider;
 
-return [
-    'debug'                        => true,
-    ConfigAggregator::ENABLE_CACHE => false,
-];
+$aggregator = new ConfigAggregator([
+    new PhpFileProvider(realpath(__DIR__) . '/autoload/{,*.}{global,local}-development.php'),
```

Comment:
File names are a bit unwieldy. I think it would be better as this:
```suggestion
    new PhpFileProvider(realpath(__DIR__) . '/autoload/{,*.}{global,local}.dev.php'),
```

Reply:
@Xerkus sorry, tooo many notifications and I totally missed this! I agree the file names are unwieldy, but that's what `laminas-development-mode` expects. Changing them would require a big BC break over there.

Reply:
Hrm... I think I'm wrong 😊 Looks like I misread the code at https://github.com/laminas/laminas-development-mode/blob/4227f01633986347bf8c038594e169f4bc4a053e/src/Enable.php#L28 all those months ago.